source/Ship.cpp

```diff
@@ -2992,6 +2996,14 @@ int Ship::RequiredCrew() const
 
 
 
+int Ship::PreferredCrew() const
+{
+	// Fall back to crew value for games saved before the preferredCrew attribute was added.
```

Comment:
This doesn't just apply to ships from older saves.
A newly bought or captured ship would also have a `preferredCrew` value of zero.